packages/core/stories/border-item.stories.tsx

```diff
@@ -8,12 +8,10 @@ export default {
     position: {
       control: { type: "select" },
     },
-    horizontalAlignment: {
-      control: { type: "select" },
-    },
-    verticalAlignment: {
-      control: { type: "select" },
+    as: {
```

Comment:
Would it work to put these argTypes globally instead of in every component?

Reply:
I guess it wouldn't harm, specially `as` and `ref` that will be in a bunch of places i would imagine